core/templates/local_vector.h

```diff
@@ -71,6 +81,14 @@ class LocalVector {
 		}
 	}
 
+	void pop_back() {
```

Comment:
It's customary to return the value instead of destructing it, so that it can be used in structures like:

```c++
while (element = list.pop_pack()) {
}
```

But this is, unfortunately, not the case with equivalent functions of other list types. So I think it would be better to keep it as-is, and address this shortcoming for all list types in a separate PR.

Reply:
I remember `std::vector::pop_back` return void because of multi-thread concern so I follow this prcatice.